docs/content/ingestion/kafka-ingestion.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: doc_page
+---
+
+# Kafka Ingestion
+
+The recommended way of ingesting data from Kafka is to use the `kafka-indexing-service` core extension (see
```

Comment:
I like the optimism but this language is a bit strong for first release :)

The language here should be telling people that this is an experimental feature, API subject to change, etc.